ReactWindows/ReactNative/ReactNative.csproj

```diff
@@ -113,7 +113,6 @@
     <Compile Include="Bridge\ReactInstance.cs" />
     <Compile Include="Bridge\DispatcherHelpers.cs" />
     <Compile Include="Bridge\JavaScriptBundleLoader.cs" />
-    <Compile Include="Bridge\NativeModuleBase.cs" />
     <Compile Include="Bridge\NativeModuleRegistry.cs" />
```

Comment:
Sounds good, thanks!

---
In reply to: [82005884](https://github.com/ReactWindows/react-native-windows/pull/740#discussion_r82005884) [](ancestors = 82005884)